data/scripts/tv.inc

```diff
@@ -42,6 +42,9 @@ EventScript_27EE9A:: @ 827EE9A
 
 EventScript_27EEA4:: @ 827EEA4
 	msgbox LittlerootTown_BrendansHouse_1F_Text_1F826F, MSGBOX_DEFAULT
+    @ This is a junk call. Its input var (VAR_0x8004) hasn't been set, and
```

Comment:
nit: tabs

Reply:
But this whole file is tabs, not spaces. Should I replace them all?

Reply:
you used spaces, when the rest of the file is tabs.  Use tabs.